src/components/FileInput/index.vue

```diff
@@ -74,27 +74,70 @@
     <div
       v-if="shouldDisplayDropbox"
       ref="dropbox"
-      :class="['drop-zone', isDragOver ? 'highlight' : null]"
+      :class="[
+        'drop-zone',
+        isDragOver ? 'highlight' : null,
+        { 'cancel-hover': selectedTabId === 'url' },
+      ]"
       @dragenter="handleDragEnter"
       @dragover="handleDragOver"
       @drop="handleDrop"
       @dragleave="handleDragLeave"
     >
-      <div class="upload-message-container">
-        <svg-icon name="upload" />
-        <p>{{ computedPlaceholderMessage }}</p>
-      </div>
+      <TabList
+        v-if="something"
+        class="file-tab-list"
+        :tab-list="tabList"
+        :selected-tab-id="selectedTabId"
+        @tab:update="handleTabUpdate"
+      />
+
+      <form
```

Comment:
`FileInput` component is used as input in form. But here we render another form inside. Please replace it with `div` for example